tests/integration-tests/test_runner.py

```diff
@@ -63,6 +63,9 @@
     "custom_awsbatch_template_url": None,
     "custom_awsbatchcli_url": None,
     "custom_ami": None,
+    "vpc-cluster": None,
```

Comment:
let's keep the same convention used above with underscores. Same for no-delete

Reply:
changed